### PR TITLE
Add `chainerx.testing.integral_dtypes`

### PR DESCRIPTION
**Pull request by niboshi**
Used in #6438.

### CODE REVIEW

File: tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py [hunk 1]
```diff
@@ -5,16 +5,34 @@
...
+        assert is_unsigned == (numpy.dtype(dtype).kind == 'u')
+
+        is_integral = dtype in chainerx.testing.integral_dtypes
+        assert is_integral == (numpy.dtype(dtype).kind in ('i', 'u'))
+
+        is_nonfloat = dtype in chainerx.testing.nonfloat_dtypes
+        assert is_nonfloat == (numpy.dtype(dtype).kind != 'f')
 
 
 def test_signed_dtypes():
```
Comment (asi1024) [inline tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py]: This test has become redundant.

File: tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py [hunk 2]
```diff
@@ -5,16 +5,34 @@
 
 
 def test_float_dtypes():
```
Comment (asi1024) [inline tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py]: The test name should be changed to something like that `test_dtypes_list`.

File: tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py (hunk 1 above)
Comment (niboshi) [reply tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py]: What do you mean?

File: tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py (hunk 2 above)
Comment (niboshi) [reply tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py]: Fixed.

File: tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py (hunk 1 above)
Comment (niboshi) [reply tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py]: Fixed.

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**